### PR TITLE
Started Android keystores chapter

### PR DESCRIPTION
**Pull request by 4ian**

### CODE REVIEW

File: android/keystore_1.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Android Keystore
+
+## Introduction on the keystores
```
Comment (4ian) [inline android/keystore_1.md]: Introduction _to_ the keystores

File: android/keystore_1.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Android Keystore
+
```
Comment (4ian) [inline android/keystore_1.md:2]: Maybe a one liner giving the motivation for this chapter ? Something in the lines, 'Your app is ready and you want to get it to your clients. In order to  do so, you need to sign your app. For that, you need to pass through the keystores'

File: android/keystore_1.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+Keystores are files containing security certificates and are primarily used to
+sign Android APK files. Usually you'll manage a single keystore that is used for
+your app in production. There is also a debug keystore that is generated by the
+Android SDK. You can check its location:
+
+```shell
+ls $HOME/.android/debug.keystore
+```
+
+If the keystore is not existing, check that you have installed the Android SDK.
```
Comment (4ian) [inline android/keystore_1.md]: doesn't exist

File: android/keystore_1.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+Keystores can be **password protected**: when it's the case, you'll be prompted for
```
Comment (4ian) [inline android/keystore_1.md]: replace it's with that's

File: android/keystore_1.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+the password when you want to use it to sign your app.
```
Comment (4ian) [inline android/keystore_1.md]: 2 times when

File: android/keystore_1.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+## Create a release keystore
+
+```shell
+keytool -genkey -v -keystore my-release-key.keystore
+-alias alias_name -keyalg RSA -keysize 2048 -validity 10000
```
Comment (4ian) [inline android/keystore_1.md:24]: Should we talk a little bit about what each parameter means? maybe not for validity which is self explanatory, but you could add a line about the my-release-key.keystore

File: android/keystore_1.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+You're done! You can send this APK to the Google Play store.
+
+## Be cautious about your keystore
+
+Keystore must be saved and **never lost**: if you lose a keystore, you won't be able
+to sign your app and release new versions on the Google Play store. In the same
+way, anyone that get access to the keystore (and its password) can use it to sign
+its own application as it if were your app.
+
+It's a good idea to never put the keystore in your git repository.
```
Comment (4ian) [inline android/keystore_1.md]: Develop this topic. Best practices, etc

File: android/keystore_1.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+zipalign -v 4 my_application.apk my_application-aligned.apk
+```
+
+You're done! You can send this APK to the Google Play store.
+
+## Be cautious about your keystore
+
+Keystore must be saved and **never lost**: if you lose a keystore, you won't be able
+to sign your app and release new versions on the Google Play store. In the same
+way, anyone that get access to the keystore (and its password) can use it to sign
```
Comment (4ian) [inline android/keystore_1.md]: gets

**Comment by 4ian** [comment]
@yleflour @yannicktian @BenjMichel @Nhacsam Merge meeeeeee

File: android/keystore_1.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
+# Android Keystore
+
+Your app is ready and you want to get it to your clients. In order to do so, you need to sign your app: for Android this is done using keystores.
+
+## Introduction to keystores
+
+Keystores are files containing security certificates and are primarily used to
+sign Android APK files. Usually you'll manage a single keystore that is used for
```
Comment (Almouro) [inline android/keystore_1.md:8]: Maybe add something explaining what it means to sign an APK?

**Comment by Almouro** [comment]
Besides that :+1:

File: android/keystore_1.md (hunk 9 above)
Comment (4ian) [inline android/keystore_1.md:8]: Done :)

**Pull request merged**